x-pack/solutions/observability/plugins/synthetics/server/tasks/sync_private_locations_monitors_task.ts

```diff
@@ -122,6 +130,7 @@ export class SyncPrivateLocationMonitorsTask {
           lastStartedAt: startedAt.toISOString(),
           lastTotalParams,
           lastTotalMWs,
+          packagePoliciesRequireReformatting,
```

Comment:
Wondering if here in the error code path we should set this to true or undefined so it forces a second attempt. Don't know. Could also result in endless errors.

Reply:
Guess we would need to have a better sense of retry-able errors in order to do that, which I don't believe we have.